operators/constellation-node-operator/config/crd/bases/update.edgeless.systems_scalinggroups.yaml

```diff
@@ -56,6 +56,10 @@ spec:
                   (used by cluster-autoscaler).
                 format: int32
                 type: integer
+              nodeGroupName:
```

Comment:
what happens with existing objects without this field? it's just empty?

Reply:
Good question. The field would remain empty. I'll need to implement a migration to patch the field in cases where the resource already exists. Thanks!